crates/common/network_spec/src/networks.rs

```diff
@@ -53,103 +59,482 @@ pub fn network_spec() -> Arc<NetworkSpec> {
     NETWORK_SPEC.get().expect("NetworkSpec wasn't set").clone()
 }
 
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq, Deserialize)]
+#[serde(rename_all = "UPPERCASE")]
 pub struct NetworkSpec {
-    pub network: Network,
-    pub genesis: Genesis,
+    pub preset_base: String,
+    pub config_name: Network,
```

Comment:
```suggestion
    #[serde(rename = "CONFIG_NAME")]
    pub network: Network,
```

I think this is cleaner, not sure why they called it config_name when it is moreso which network it is